mattrubin/Authenticator
# Fastlane Icon Generation

Update the icon generation script to:
- work with the latest Inkscape
- run as a fastlane task instead of a build phase
- create the `app_icon.png` used by fastlane's `deploy`

Also, modify the `beta` and `release` lanes to ensure the generated assets are updated.
